app/views/simplified-account/settings/stripe-details/bank-account/index.njk

```diff
@@ -77,7 +77,6 @@
     document.getElementById('bank-account-form').addEventListener('submit', function () {
       submitButton.setAttribute('disabled', 'true')
       submitButton.setAttribute('aria-disabled', 'true')
-      submitButton.setAttribute('class', 'govuk-button govuk-button--disabled')
```

Comment:
I presume that the behaviour is still the same after removing this?

Reply:
Yes - I tested this by creating a little test.
Good questions.